### PR TITLE
Creates and displays a google map

### PR DESCRIPTION
**Pull request by rysimone**

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -33,11 +33,6 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
-    <dependency>
-      <groupId>com.google.appengine</groupId>
```
Comment (tjgoog) [inline portfolio/pom.xml:37]: Is this still needed for other APIs like login or datastore?

File: portfolio/src/main/webapp/maps.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+const mapsKey = config.MAPS_KEY;
+const surfSpots = ['pv', 'tpz', 'bw', 'lf', 'nmbp', 'porto'];
```
Comment (tjgoog) [inline portfolio/src/main/webapp/maps.js:2]: These seem to be unused

File: portfolio/src/main/webapp/maps.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+window.onload = createScriptTagForMap();
+
+
+function createMap() {
+    const map = new google.maps.Map(
+        document.getElementById('map-container'),
+        {center: {lat:33.835, lng: -118.390}, zoom: 12});
```
Comment (tjgoog) [inline portfolio/src/main/webapp/maps.js:9]: Could you add a comment describing the default map center? E.g. "Default to southern California" or wherever.

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -37,4 +37,16 @@
 
 #comment-text-field {
   width: 600px;
+}
+
+#map-container {
+  height: 600px;
+}
+
+.beach {
```
Comment (tjgoog) [inline portfolio/src/main/webapp/style.css:46]: I think the `beach` class isn't currently used

**Comment by tjgoog** [comment]
Oops, I forgot that we're back to both reviewing now, so please don't merge until Megan has gotten to take a look too.

File: portfolio/pom.xml (hunk 1 above)
Comment (rysimone) [reply portfolio/pom.xml:37]: I accidentally added that dependency twice so I was removing one of them.

**Pull request merged**